Physik.tex

```diff
@@ -31,28 +40,503 @@
+\subsection{Kraft, Energie, Impuls}
+\subsubsection{Kraft}
+Federkraft: $F_F=-k(x-x_0)$ \\
+Gewichtskraft: $F_G=mg$ \\
+$F_N=mg\cos \alpha$ \\
+$F_H=mg\sin \alpha$ \\
+Zentripetalkraft: $F_Z=m\cdot \frac{v^2}{r}=m\omega^2r=p\omega$ \\
+Reibungskraft: $F_R=\mu F_{(N)}=\mu m g \cos \alpha$
+\subsubsection{Energie}
+$E_{\ir kin}=\frac{1}{2}mv^2=\frac{p^2}{2m}$\\
+$E_{\ir pot}=mgh$\\
+$E_{F}=\frac{1}{2}k(x-x_0)^2$\\
+$E_{\ir rot}=\frac{1}{2}I\omega^2$
+\subsubsection{Arbeit}
+$W=\Delta E=F\cdot s \cdot \cos \alpha$
+\subsubsection{Leistung}
+$P=
+\underbrace{\frac{\Delta W}{\Delta t}}_{\text{mittlere Leistung}} 
+= \, \underbrace{\frac{F\cdot \Delta s}{\Delta t}=F\cdot r}_{\text{momentane Leistung}}$\\
+$\eta=\frac{E_{\it nutz}}{E_{\ir zu}}=\frac{P}{P}$
+\subsubsection{Impuls}
+$p_{\ir vorher}=p_{\ir nachher}$\\
+$P=mv$\\
+Mittlere Kraft: $\overline{F}=\frac{\Delta p}{\Delta t}$\\
+Elastischer Stoß: $m_1v_1+m_2v_2=m_1u_1+m_2u_2$\\
+Unelastischer Stoß: $m_1v_1+m_2v_2=(m_1+m_2)u$
+\subsubsection{Raketengleichung}
+$F=ma=\frac{\diff m}{\diff t}v_a+F_{\ir ext}$\\
+$\vec{v}=\vec{v_0}+\vec{v_a}\ln | \frac{M_G}{m} | $\\
+$m_0=M+m_a$
+\subsection{Planetenbewegungen}
+\subsubsection{Gravitation}
+$U_{\ir pot}=-G\frac{m_E\cdot m}{r} \qquad r>r_E$\\
+$\vec{F}_G=G\frac{m_1m_2}{r^2}\cdot \vec{e_r}$\\
+$g=G\cdot \frac{m}{r^2}$\\
+$W_G=Gm_1m_2(\frac{1}{r_1}-\frac{1}{r_2})$\\
+$F_G \, || \, r \rightarrow M=r\times F=0$\\
+
+Satellit-Erde: $\frac{1}{2}m_Sv^2=G\frac{m_Sm_E}{R_E}$ \\
+Stabile Umlaufbahn: $\omega_S=\sqrt{\frac{G\cdot m_E}{R_E^3}}\qquad v=\omega R_E$\\
+Geostat. Umlaufbahn: $r_g=\sqrt[3]{G\cdot m_E \cdot \frac{T_g^2}{4\pi^2}} \qquad h_g=r_g-R_E$
 
-% Section
+\subsubsection{Keplersche Gesetze}
+\begin{sectionbox}
+\begin{itemize}
+\item[1.] Keplersches Gesetz: Planetenbahnen sind Ellipsen um Stern in einem der beiden Brennpunkte
+\item[2.] Keplersches Gesetz: In gleicher Zeit wird die gleiche Fläche an einer Bahn aufgespannt\\$\frac{dA}{dt} = \frac{1}{2} \vec {r} \vec {v} sin \alpha = \frac{1}{2}\vec {r} \times \vec {v} = \frac{1}{2}m|\vec {L}| \Rightarrow$ Der Drehimpuls ist zeitlich konstant
+\item[3.] Keplersches Gesetz: $\frac{T_1^2}{T_2^2} = \frac{a_1^3}{a_2^3}$ mit T: Umlaufzeit, a: Große Halbachse
+\end{itemize}
+\end{sectionbox}
+\subsection{Drehungen}
+\subsubsection{Drehmoment}
+$M=F\times r=F\cdot r\cdot \sin \alpha$\\
+$M=I\cdot \dot{\omega}=\frac{\diff L}{\diff t}$
+\subsubsection{Trägheitsmoment}
+Zylindermantel: $J = mr^2$ (Drehachse ist Körperachse)\\
+Hohlzylinder: $J = \frac{1}{2}m(r_a^2+r_i^2)$ (Drehachse ist Körperachse)\\
+Massive Kugel: $J = \frac{2}{5}mr^2$ (Drehachse durch Mittelpunkt)\\
+\subsubsection{Drehimpuls}
+$L=r\times p=I\cdot \omega = mrv$
+\subsection{Corioliskraft}
+%\begin{emphbox}
+%Beispielsweise Textcolor anstatt emphbox verwenden und alle wichtige Formell mit rot bzw. einer anderen Farbe schreiben.
+\textcolor{red}{$\vec{F_c}=2m(v\times \omega)= 2m v w \cdot \sin{(v\angle \omega)}$}\\
```

Comment:
Let's remove the textcolor and treat all formulas the same then

Reply:
I agree, let's do it so.